pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -903,6 +903,33 @@ func Convert_v1_Rng_To_api_Rng(_ *v1.Rng, rng *api.Rng, c *ConverterContext) err
 	return nil
 }
 
+func Convert_v1_Usbredir_To_api_Usbredir(vmi *v1.VirtualMachineInstance, domainDevices *api.Devices, _ *ConverterContext) (bool, error) {
+	clientDevices := vmi.Spec.Domain.Devices.ClientPassthrough
+
+	// Default is to have USB Redirection disabled
+	if clientDevices == nil {
+		return false, nil
+	}
+
+	// Note that at the moment, we don't require any specific input to configure the USB devices
+	// so we simply create the maximum allowed dictated by v1.UsbClientPassthroughMaxNumberOf
+	redirectDevices := make([]api.RedirectedDevice, v1.UsbClientPassthroughMaxNumberOf)
+
+	for i := 0; i < v1.UsbClientPassthroughMaxNumberOf; i++ {
+		path := fmt.Sprintf("/var/run/kubevirt-private/%s/virt-usbredir-%d", vmi.ObjectMeta.UID, i)
```

Comment:
I don't think that the ` vmi.ObjectMeta.UID` needs to be part of the path. `/var/run/kubevirt-private` is already a private directory inside a specific `virt-launcher` container.

Since the PR exists for quite some time, this may have indeed been needed initially.

Reply:
Looking at it now, it is indeed not a must to put it under VM's UID folder but the other subresources share the same path, including `.../virt-vnc` and `.../virt-serial0` so I'd let it stay in this UID folder for consistency or consider changing them all to another path.

Reply:
Works for me to leave it consistent.